enterprise/websvc.wsitconf/licenseinfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+  <!--
+ 
+     Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+     Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+   
+ -->
+  <licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/DerivedKeyPasswordValidator.template</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/Saml11HOKCallbackHandler.template</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/Saml11SVCallbackHandler.template</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/Saml20HOKCallbackHandler.template</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/Saml20SVCallbackHandler.template</file>
+        <file>src/org/netbeans/modules/websvc/wsitconf/resources/templates/sts_schema.template</file>
+        <license ref="Apache-2.0-ASF"/>
+        <comment type="COMMENT_UNSUPPORTED"/>
```

Comment:
This is not correct - comments are actually supported, but at this point we choose to allow templates without ASF headers. So please use type='TEMPLATE_MINIMAL_IP'.

I noticed, that `sts.template` and `sts13.template` both contain an ASF header, while the other files don't. I would remove the headers from these two files and add these to this fileset, to unify that.